lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -93,6 +93,33 @@ def initialize(name:)
     class RequiredParameter < Parameter
     end
 
+    class Attribute < Entry
```

Comment:
I'm not sure about the name. It seems we're more interested in the accessor rather than the real attribute that is behind. Do we plan to store in the index the location of an attribute initialization like `@foo = "foo"`? In which case we would need to rename this class anyway.

We kinda see this naming problem when you set the symbol kind: `PROPERTY`, I would expect the property to be the accessors and the real attribute to be a `FIELD`.

What do you think of renaming it as `Accessor` or maybe `AttributeAccessor`?

Reply:
That's fair. We will eventually want to support instance variables. I renamed to `Accessor`.